src/languages/es.js

```diff
@@ -139,6 +139,13 @@ export default {
         updateApp: 'Actualizar app',
         updatePrompt: 'Existe una nueva versión de esta aplicación.\nActualiza ahora or reinicia la aplicación más tarde para recibir la última versión.',
     },
+    deeplinkWrapper: {
+        // TODO: translate to spanish
+        launching: 'Launching Expensify',
+        redirectedToDesktopApp: 'We\'ve redirected you to the desktop app.',
+        youCanAlso: 'You can also',
+        openLinkInBrowser: 'open this link in your browser',
+    },
```

Comment:
@NikkiWines We need Spanish copy, can you add `wait for copy `label

Reply:
I think the label has to be applied to the GH issue for it to work

Reply:
I think the `Waiting for copy` label on the issue assigns someone from our Marketing team which wouldn't necessarily be helpful. Looks like @iwiznia has 👍 your translations @aldo-expensify so we're all good! 

Thanks @iwiznia 🙌